translations/en-us.yaml

```diff
@@ -3343,16 +3343,51 @@ viewEditProject:
     Anybody with access to the API/UI acts as an admin and will be able to use any environment.
 
 volumesPage:
+  title: Volume
   noMatch: No volumes match the current search
+  banner:
+    driver: Driver
+    size: Size (Mb)
+    stack: Stack
+    uri: URI
+    perCont: On Container
+    host: Host
   scope:
     label: Scope
     stack: Stack
     container: Per Container
     standalone: Environment
   driver:
     label: Driver
+  driverOptions:
+    title: Driver Options
+    detail: Map of key value pairs representing driver options
+    labels:
+      key: Key
+      value: Value
+      noData: No Options
+    status: |
```

Comment:
Add a new section to `pagination` (with support for `pages` variable too, don't just copy this one) instead of defining here.

Then re-use that as the `pagingLabel` on the `sortable-table` for the paging control to show the right word.